README.md

```diff
@@ -80,76 +80,80 @@ This example demonstrates how to transcribe a local audio file:
 import WhisperKit
 
 // Initialize WhisperKit with default settings
-Task {
-   let pipe = try? await WhisperKit()
-   let transcription = try? await pipe!.transcribe(audioPath: "path/to/your/audio.{wav,mp3,m4a,flac}")?.text
-    print(transcription)
-}
+let pipe = try await WhisperKit()
+// Transcribe the audio file
+let transcription = try await pipe.transcribe(audioPath: "path/to/your/audio.{wav,mp3,m4a,flac}")?.text
+// Print the transcription
+print(transcription)
```

Comment:
removed not needed `Task`, decided to remove `?` from `try ?`. When the potential user does copy paste from the readme I think it's better when the error is not ignored